effectful-core/src/Effectful.hs

```diff
@@ -24,13 +24,31 @@ module Effectful
   , withEffToIO
 
   -- * 'Effect' handlers
+  , EffectStyle
+  , DynamicEffect
+  , StaticEffect(..)
 
   -- ** Sending operations to the handler
   , send
 
   -- ** Basic handlers
-  , interpret
-  , reinterpret
+  , DynamicEffectHandler
```

Comment:
Why did you rename functions for dynamic effects? The suffix `Dynamic` is generally redundant, there is no sensible function that would be named `interpretStatic`.

Reply:
I think it makes it more obvious what the function is about. 

Yes, there is no corresponding function for static effects, but I don't think having suffix makes it a lot easier for a person to initially grasp the interface. 

---

(I will not fight hard on this, just thought this naming would be more obvious)

Reply:
Please introduce minimal changes that only introduce safety, no API renaming.